app/repository/StageRepositoryImpl.java

```diff
@@ -12,6 +12,7 @@
 import javax.persistence.TypedQuery;
 import java.util.Calendar;
 import java.util.List;
+import java.util.SimpleTimeZone;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.SimpleTimeZone'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)